host/sgx/sgxload.c

```diff
@@ -918,16 +919,39 @@ oe_result_t oe_sgx_initialize_enclave(
             (void*)&launch_token,
             sizeof(info.EInitToken)));
 
-        if (!InitializeEnclave(
+        bool result = false;
```

Comment:
Check that compilation on Windows doesn't complain about the implicit cast from BOOL to bool. Probably just easier to use the Windows type. 

You're also shadowing the existing `result` variable in this function, please rename it:

```c
BOOL success = FALSE;
```

Reply:
ok